README.md

```diff
@@ -7,38 +7,70 @@ It is very much a work in progress.
 Getting Started
 ---------------
 
-You will need to install and configure [calyx][] and the python bindings for [odgi]. Instructions for installing odgi can be found [here](https://odgi.readthedocs.io/en/latest/rst/installation.html). If you compile odgi from its source, you will need to [edit your python path](https://odgi.readthedocs.io/en/latest/rst/binding/usage.html) to use the python bindings. Since this will only give you the odgi bindings for the most recent version of python, you may need to download the `.so` files from [bioconda](https://odgi.readthedocs.io/en/latest/rst/binding/usage.html) and add them to your `PYTHONPATH` instead.
+First, clone the repo using 
+```git clone https://github.com/cucapra/calyx-pangenome.git```
+Then follow the instructions below to set up `calyx` and `odgi`.
 
-You will need to fetch some useful inputs by typing `make fetch`.
-This downloads some simple [GFA][] data files into the `./test` directory.
+### Installing Dependencies
 
-If you want to build odgi graph files from the GFA files, try something like `make k.og`.
+#### Calyx
+Follow these [instructions](https://docs.calyxir.org/) to install calyx. You will need to configure the calyx driver, `fud`. We recommend using the native calyx interpreter, so once `fud` is set up, run
+```
+fud config stages.interpreter $ABSOLUTE_PATH_TO_CALYX_REPO/target/debug/interp
+```
+where `$ABSOLUTE_PATH_TO_CALYX_REPO` is the absolute path to the root directory. For example, if you downloaded calyx in `/Users/username/project`, you would run `fud config stages.interpreter /Users/username/project/target/debug/interp`.
+
+#### Odgi
+
+You will need to install the python bindings for [odgi]. Instructions for installing odgi can be found [here](https://odgi.readthedocs.io/en/latest/rst/installation.html). If you compile odgi from its source, you will need to [edit your python path](https://odgi.readthedocs.io/en/latest/rst/binding/usage.html) to use the python bindings.
+
+To verify that you have the python bindings, open up a python shell and try `import odgi`. If this doesn't work, you can try a different method of installation, or you can download the `.so` files from [bioconda](https://anaconda.org/bioconda/odgi/files) for the version of python you are running (`python3 --version`) and add them to your `PYTHONPATH` instead.
```

Comment:
Feels like we're leaving them in the lurch, asking them to flail around and maybe also fix versions. Maybe we'll learn more ourselves in the weeks to come, but eventually it would be nice to say, "Here is the odgi install guide; they give three options but we we recommend that you chose [the Bioconda route](https://odgi.readthedocs.io/en/latest/rst/installation.html#bioconda)." Or something like that.

For the .so file, it may be worth giving a tiny example. So like, "If the result of `python3 --version` is 3.7, fetch `odgi...py37....tar.bz2`.